setup.cfg

```diff
@@ -27,3 +27,7 @@ github_user = mozilla
 github_project = ichnaea
 bugzilla_product = Location
 bugzilla_component = General
+
+[tool:paul-mclendahand]
+github_user = mozilla
+github_project = ichnaea
```

Comment:
I get a little itchy with the duplicated key-value pairs, but I don't believe there is an existing ``setup.cfg`` standard to hook into, and I'm not excited about wading into ``setuptools`` just yet. Until then, explicit is better than implicit.